far/copy.cpp

```diff
@@ -797,7 +797,7 @@ ShellCopy::ShellCopy(
 			msg(Link? lng::MCMLTargetIN : lng::MCMLTargetTO));
 	}
 
-	CopyDlg[ID_SC_TARGETTITLE].strData=strCopyStr;
+	CopyDlg[ID_SC_TARGETTITLE].strData = std::move(strCopyStr);
```

Comment:
It is a valid optimisation, but:
- It makes the code harder to support: now it relies on an *assumption* (not a statically proven fact) that `strCopyStr` is not used anywhere below this line. It could be the case right now, but tomorrow someone could use `strCopyStr` for something else and it might even work for some time due to SSO and then suddenly break.
- It's not a hot path, just showing the copy dialog *once*, so we won't see any effect of it whatsoever.

Reply:
@alabuzhev, clearly this may prevent further code modification, but the IDE returns a warning when reused after `std::move()`. Most developers disable compiler alerts, so I agree that this will make it difficult to change the code, but when the code is final, you will have to use `std::move()`.

Reply:
> IDE returns a warning when reused after `std::move()`

That would've been great, but, [unfortunately, no](https://godbolt.org/z/P4zzbPK1W).